src/Model/Behavior/ShadowTranslateBehavior.php

```diff
@@ -378,6 +378,9 @@ public function groupTranslations($results)
     {
         return $results->map(function ($row) {
             $translations = (array)$row['_i18n'];
+            if (empty($translations) && $row->get('_translations')) {
```

Comment:
I'll happily change it - I just copied it directly from https://github.com/cakephp/cakephp/blob/master/src/ORM/Behavior/TranslateBehavior.php#L460 :)